packages/core/test/popover/popoverTests.tsx

```diff
@@ -76,6 +83,51 @@ describe("<Popover>", () => {
             assert.isTrue(warnSpy.calledWith(Errors.POPOVER_WARN_DOUBLE_CONTENT));
             warnSpy.restore();
         });
+
+        it("warns if attempting to open a popover with empty content", () => {
+            const warnSpy = sinon.stub(console, "warn");
+            shallow(
+                <Popover content={null} isOpen={true}>
+                    {"target"}
+                </Popover>,
+            );
+            assert.isTrue(warnSpy.calledWith(Errors.POPOVER_WARN_EMPTY_CONTENT));
+            warnSpy.restore();
+        });
+
+        it("warns if backdrop enabled when rendering inline", () => {
+            const warnSpy = sinon.stub(console, "warn");
+            shallow(
+                <Popover hasBackdrop={true} inline={true}>
+                    {"target"}
+                    {"content"}
+                </Popover>,
+            );
+            assert.isTrue(warnSpy.calledWith(Errors.POPOVER_WARN_MODAL_INLINE));
+            warnSpy.restore();
+        });
+
+        describe("throws error if backdrop enabled with non-CLICK interactionKind", () => {
+            runErrorTest("HOVER", PopoverInteractionKind.HOVER);
+            runErrorTest("HOVER_TARGET_ONLY", PopoverInteractionKind.HOVER_TARGET_ONLY);
+            runErrorTest("CLICK_TARGET_ONLY", PopoverInteractionKind.CLICK_TARGET_ONLY);
+
+            it("doesn't throw error for CLICK", () => {
+                assert.doesNotThrow(() => (
+                    <Popover hasBackdrop={true} interactionKind={PopoverInteractionKind.CLICK} />
+                ));
+            });
+
+            function runErrorTest(testName: string, interactionKind: PopoverInteractionKind) {
```

Comment:
you can get the name of a `number` enum value like `PopoverInteractionKind[interactionKind]`

Reply:
Thought about that. Will do.

Reply:
Used `keyof typeof PopoverInteractionKind`